R/install_livy.R

```diff
@@ -0,0 +1,70 @@
+#' Install Livy
+#'
+#' Automatically download and install \href{http://livy.io/}{\samp{livy}}.
+#' \samp{livy} provides a REST API to Spark.
+#'
+#' @param spark_home The path to a Spark installation. The downloaded and
+#'   installed version of \samp{livy} will then be associated with this Spark
+#'   installation.
+#' @param livy_version The version of \samp{livy} to be installed. Currently,
+#'   only the beta version, 0.2.0, is available.
+#' @param hadoop_conf_dir Optional; the path to a Hadoop configuration directory.
+#' @export
+livy_install <- function(spark_home = Sys.getenv("SPARK_HOME"),
```

Comment:
I think we want to replace `spark_home` for a `version` parameter to specify the spark version. When null, we can get the `spark_home` using `spark_home_dir()` instead and also pass the version from the parameter down as `spark_home_dir(version = version)`. That way, users don't need to specify a `SPARK_HOME` explicitly. I think a `version` parameter is better since we would also want to check against supported Spark versions, see comment bellow. Other than that, this looks great!